dev_guide/application_lifecycle/new_app.adoc

```diff
@@ -86,29 +80,33 @@ xref:language-detection[detects an appropriate language builder image].
 If a *_Jenkinsfile_* exists in the root or specified context directory of the
 source repository when creating a new application, {product-title} generates a
 xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[`*Pipeline*`
-build strategy]. 
+build strategy].
 ifndef::openshift-online[]
 Otherwise, if a *_Dockerfile_* is found, {product-title}
 generates a
 xref:../../architecture/core_concepts/builds_and_image_streams.adoc#docker-build[`*Docker*`
-build strategy]. 
+build strategy].
 endif::[]
 Otherwise, it generates a
 xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[`*Source*`
 build strategy].
 
 You can override the build strategy by setting the `--strategy` flag to either
 ifndef::openshift-online[]
-`docker`, 
+`docker`,
 endif::[]
 `pipeline` or `source`.
 
-====
+ifndef::openshift-online[]
```

Comment:
Ooohhh I thought this was a typo, but google told me it's an actual thing. Good to know!